src/components/icon/icon.md

```diff
@@ -1,18 +1,21 @@
 Basic usage:
 
 ```html
-<Icon.Trash fill="tomato" stroke="tomato" width={48} height={48} />
+<Icon.Trash fill="tomato" stroke="tomato" width="{48}" height="{48}" />
```

Comment:
There should not be double quotation marks here, around `{48}`?

Reply:
Is it because of the `html` type of the verbatim?

Reply:
this was just prettier doing whatever prettier does...